pkg/specgen/generate/container.go

```diff
@@ -303,8 +303,8 @@ func FinishThrottleDevices(s *specgen.SpecGenerator) error {
 			if err := unix.Stat(k, &statT); err != nil {
 				return err
 			}
-			v.Major = (int64(unix.Major(uint64(statT.Rdev))))
```

Comment:
I think we need this. Stat_t.Rdev is not uint64 on every architecture, compare https://github.com/containers/psgo/pull/77/files

Some problem with the other places. I am not sure if this allies to all fields in the stat struct but Rdev definitely one where we have to convert.

Reply:
Updated :heavy_check_mark: